third_party/move/move-compiler/src/inlining/translate.rs

```diff
@@ -1099,12 +1099,20 @@ fn get_params_from_decls(decls: &LValueList) -> Vec<Symbol> {
         .value
         .iter()
         .flat_map(|lv| match &lv.value {
-            LValue_::Var(v, _) => vec![v.0.value],
-            LValue_::Ignore => vec![],
+            LValue_::Var(v, _) => vec![Some(v.0.value)],
+            LValue_::Ignore => vec![None], // placeholderfor "_"
             LValue_::Unpack(_, _, _, fields) | LValue_::BorrowUnpack(_, _, _, _, fields) => {
-                fields.iter().map(|(_, x, _)| *x).collect()
+                fields.iter().map(|(_, x, _)| Some(*x)).collect()
             },
         })
+        .enumerate()
+        .map(|(idx, opt_sym)| {
+            if let Some(sym) = opt_sym {
+                sym
+            } else {
+                Symbol::from(format!("_${}", idx))
```

Comment:
At some point, we may want to make a "factory" for various compiler mangled symbols, so that they don't clash with each other in unexpected ways.

Reply:
Maybe I'll add it here, as I seem to still have a few issues to resolve.

Reply:
I just abstracted it locally, so that renaming within the inliner is at least consistent.